tests/test_preprocessing.py

```diff
@@ -114,6 +114,13 @@ def test_pipeline_stopwords(self):
         pipeline = [preprocessing.lowercase, preprocessing.remove_stopwords]
         self.assertEqual(preprocessing.clean(s, pipeline=pipeline), s_true)
 
+    def test_pipeline_default(self):
+        s = pd.Series(
+            "Amazon! < br />< br /> If I was going to order any soft drink online, it would be Diet Coke with Lime"
```

Comment:
Can you please add an example for each pipe component? As the default clean pipeline is:

1. fillna
2. lowercase
3. remove digits 
4. ...

It would be great that in the `test_pipeline_default ` we test for all these conditions, i.e test against a pd.NA cell, with UPPERCASED word, some 9numbers, etc.